weblate/auth/models.py

```diff
@@ -114,12 +114,12 @@ class Group(models.Model):
     )
 
     project_selection = models.IntegerField(
-        verbose_name=_('Project selection'),
```

Comment:
This is confusing - this field determines how project selection (which is just next field) works.

Reply:
Could it be "Choose language"?

Reply:
IMHO no, take a look at admin interface screenshot:

![screenshot-2018-6-17 change group weblate administration](https://user-images.githubusercontent.com/212189/41505832-798bbdde-7211-11e8-8c52-b907a028e727.png)